example-imageGestures/addons.make

```diff
@@ -1 +1,2 @@
 ofxMultitouch
+ofxMultitouch
```

Comment:
In all honestly I don’t remember anything about the addon.make file, so I’m working off the [two sentences in the docs](https://openframeworks.cc/learning/01_basics/how_to_add_addon_to_project/#:~:text=a%20file%20called-,addons.make,-.%20It%20should%20be) that covers it - is the duplication here and in the other two addons.make files intentional?

Reply:
The additional line isn't required. ProjectGenerator starts adding multiple copies of add-ons as it's run multiple times, so this second line can be removed.